test/specs/basic/ignore-joiners-synonyms-diacritics.js

```diff
@@ -0,0 +1,29 @@
+/*!***************************************************
+ * mark.js
+ * https://github.com/julmot/mark.js
+ * Copyright (c) 2014–2016, Julian Motz
+ * Released under the MIT license https://git.io/vwTVl
+ *****************************************************/
+"use strict";
+describe("basic mark with ignoreJoiners with synonyms and diacritics", function () {
+    var $ctx;
+    beforeEach(function (done) {
+        loadFixtures("basic/ignore-joiners-synonyms-diacritics.html");
+
+        $ctx = $(".basic-ignore-joiners-synonyms-diacritics");
+        new Mark($ctx[0]).mark(["Dolor", "Aliquam", "Sed", "Lorèm ipsum"], {
+            "separateWordSearch": false,
+            "synonyms": {
+                "Sed": "justø",
+                "Dolor": "Ãmet"
+            },
+            "caseSensitive": true,
```

Comment:
![244d353bfb0b3ad1854098555021b8d7a439b5bb1d66488313c1caaf2938a3ef](https://cloud.githubusercontent.com/assets/8310677/18383748/2393513a-7687-11e6-8388-9dec504e675d.jpg)

First of, dafuq??? It was 5am when you commented? 

To answer your question; yes that would be okay.